client/src/footer/Footer.jsx

```diff
@@ -0,0 +1,41 @@
+import React, {
+    Component
+} from "react";
+import logo_HYF from '../logo_HYF.svg';
+import logo from '../logoVox.png'
+import { BsEnvelope } from 'react-icons/bs';
+import { FaGithub } from 'react-icons/fa';
+
+class Footer extends Component {
+    state = {
+        email: "mailto:voxbox@gmail.com",
+        repoUrl: "https://github.com/gelilaa/VoxBox",
+    };
+
+    styles = {
+        color: 'white'
+    };
+
+    render() {
+        return ( 
+            <div style = {{backgroundColor:'#397e6b', color:'white'}}className = "footer text-white  row p-2 fixed-bottom" >
+            <div className = "col-md-8" >
+            <ul style={{listStyleType: 'none'}}>
+            <li className = "card-title">Contact VoxBox</li>
+            <li><FaGithub /><a href = { this.state.repoUrl }  style={this.styles} > https: //github.com/gelilaa/VoxBox < /a></li>
```

Comment:
there is a gap  < /a> here that make it to give an error

Reply:
Thanks, I have just fixed and pushed